mapbox_vector_tile/encoder.py

```diff
@@ -56,13 +56,28 @@ def transform(shape, func):
     raise ValueError('Unknown geometry type, "%s"' % shape.type)
 
 
+def on_invalid_geometry_raise(shape):
+    raise Exception('Invalid geometry: %s' % shape.wkt)
+
+
+def on_invalid_geometry_ignore(shape):
+    return None
+
+
+def on_invalid_geometry_make_valid(shape):
+    if shape.type in ('Polygon', 'MultiPolygon'):
+        shape = shape.buffer(0)
```

Comment:
Sorry, by "quantize" I meant "round" - that the `buffer(0)` process might introduce new points at non-integer coordinates.

I think we might also need to recurse into `MULTI*` types - it's possible that only one part of a multi-geometry is invalid and we wouldn't want to drop the whole thing. Yikes, this is getting complicated.